hudi-sync/hudi-sync-common/src/main/java/org/apache/hudi/sync/common/util/Parquet2SparkSchemaUtils.java

```diff
@@ -133,7 +154,7 @@ private static String convertPrimitiveType(PrimitiveType field) {
 
   private static String convertGroupField(GroupType field) {
     if (field.getOriginalType() == null) {
-      return convertToSparkSchemaJson(field);
+      return convertToSparkSchemaJson(field, Arrays.asList());
     }
```

Comment:
Does this mean composition data type is not supported?

Reply:
yes, currently in hive sync we don't have the composition data type `List<FieldSchema>`  is flat